services/graph/pkg/service/v0/service.go

```diff
@@ -281,6 +299,7 @@ func NewService(opts ...Option) Service {
 					})
 				})
 			})
+			r.Get("/applications/{applicationID}", svc.GetApplication)
```

Comment:
- [ ] implement listing without knowing the id to discover the apps and their approles